### PR TITLE
Contrast 13226 org stats

### PR DESCRIPTION
**Pull request by kaitlynmortimer**

### CODE REVIEW

File: content/user/ui/report/Org-stats.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+tags: "TeamServer licenses applications servers organizations report"
+-->
+
+For a summary of administrative statistics on your Contrast installation, look no further than your Organization Statistics. 
+
+Go go to the Reports page via the User Menu to find information about licensing and breakdowns across various data points for onboarded applications; deployed servers and how they’re being utilized; and licensing to understand utilization and expirations. A collection of filters and charts let you decide which data you want to see and how you want to see it. 
+
+<a href="assets/images/org-stats-reports-page.png" rel="lightbox" title="Organization statistics on Reports page"><img class="thumbnail" src="assets/images/org-stats-reports-page.png"/></a>
+
+In the Licenses chart, view the number of overall licenses for Assess and Protect, as well as the the number of unlicensed applications and servers that exist in your organization. Click the application count to go to the Trial/Unlicensed quick view in the Applications page.
```
Comment (jday3) [inline content/user/ui/report/Org-stats.md]: Discussed with Terra, lets update "Trial/Unlicensed" to just "Unlicensed"

File: content/user/ui/report/Org-stats.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+In the next chart, view active applications. The inner ring designates the breakdown by language; choose the breakdown of the outer ring by selecting **Technology** or **Grade** in the dropdown menu. 
+
+Last, but not least, view the Servers chart. Select **Container** or **Environment** in the dropdown menu to choose how the numbers are analyzed. 
+
+For more details on each category, select the **View** link under each heading. 
+
+## Licenses
+
+The Licenses tab features a Consumption trend chart of data for the past 12 months. Hover over a data point on the Assessment or Protection trend lines to see how may licenses were available that month. The thermometer chart below shows the total number of licenses purchased compared to the number being used. The timeline of shows how many licenses are about to expire on given dates. For a different view of the data, take a look at the circular charts with breakdowns by fraction and percentage for Assessment and Protection.
```
Comment (jday3) [inline content/user/ui/report/Org-stats.md]: Typo in "may" - also, let's update this phrase to "see how many licenses were being utilized that month"

Comment (jday3) [inline content/user/ui/report/Org-stats.md]: "The timeline of shows" should be "The timeline shows"

File: content/user/ui/report/Org-stats.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+If you have no Protection licenses, Contrast alerts you to the count of unlicensed servers.
+
+<a href="assets/images/org-stats-licenses.png" rel="lightbox" title="Organization statistics for licenses"><img class="thumbnail" src="assets/images/org-stats-licenses.png"/></a>
+
+<!-- Couldn't fit Assessment AND Protection view in browser window-->
+
+## Applications
+
+In the Applications tab, the Status thermometer chart shows the total number of applications broken down by the number that are licensed, unlicensed and archived. Click on the total number of onboarded applications to go to the Applications page for more details on each one. Contrast also remdinds you of how many licenses are available in your organization.
```
Comment (jday3) [inline content/user/ui/report/Org-stats.md]: remdinds -> reminds

**Comment by jday3** [comment]
Aside from the few comments I've made, this looks great! 🏰

File: content/user/ui/report/Org-stats.md (hunk 3 above)
Comment (kaitlynmortimer) [inline content/user/ui/report/Org-stats.md]: Gahhh...I always forget to turn on spellcheck

File: content/user/ui/report/Org-stats.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+<!--
+title: "Organization Statistics"
+description: "Overview of organization statistics charts"
+tags: "TeamServer licenses applications servers organizations report"
+-->
+
+For a summary of administrative statistics on your Contrast installation, look no further than your Organization Statistics. 
+
+Go go to the Reports page via the User Menu to find information about licensing and breakdowns across various data points for onboarded applications; deployed servers and how they’re being utilized; and licensing to understand utilization and expirations. A collection of filters and charts let you decide which data you want to see and how you want to see it.
```
Comment (terra4383) [inline content/user/ui/report/Org-stats.md]: Remove an instance of "go"

File: content/user/ui/report/Org-stats.md (hunk 1 above)
Comment (terra4383) [inline content/user/ui/report/Org-stats.md]: remove instance of "the"

File: content/user/ui/report/Org-stats.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+For a summary of administrative statistics on your Contrast installation, look no further than your Organization Statistics. 
+
+Go go to the Reports page via the User Menu to find information about licensing and breakdowns across various data points for onboarded applications; deployed servers and how they’re being utilized; and licensing to understand utilization and expirations. A collection of filters and charts let you decide which data you want to see and how you want to see it. 
+
+<a href="assets/images/org-stats-reports-page.png" rel="lightbox" title="Organization statistics on Reports page"><img class="thumbnail" src="assets/images/org-stats-reports-page.png"/></a>
+
+In the Licenses chart, view the number of overall licenses for Assess and Protect, as well as the the number of unlicensed applications and servers that exist in your organization. Click the application count to go to the Trial/Unlicensed quick view in the Applications page. 
+
+In the next chart, view active applications. The inner ring designates the breakdown by language; choose the breakdown of the outer ring by selecting **Technology** or **Grade** in the dropdown menu.
```
Comment (terra4383) [inline content/user/ui/report/Org-stats.md]: "...choose **what you'd like to compare it to in** the outer ring by selecting..."

Comment (terra4383) [inline content/user/ui/report/Org-stats.md]: "In the next chart, get a glimpse into your active applications."

File: content/user/ui/report/Org-stats.md (hunk 1 above)
Comment (terra4383) [inline content/user/ui/report/Org-stats.md]: "Clicking the application count will navigate you to the Unlicensed quick view on the Applications page."

File: content/user/ui/report/Org-stats.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
+<!--
+title: "Organization Statistics"
+description: "Overview of organization statistics charts"
+tags: "TeamServer licenses applications servers organizations report"
+-->
+
+For a summary of administrative statistics on your Contrast installation, look no further than your Organization Statistics.
```
Comment (terra4383) [inline content/user/ui/report/Org-stats.md]: For some reason this doesn't sound like our article voice to me. Can we try rephrasing? Example: "Organization Statistics provides robust and comprehensive visibility into your most crucial organization data."

File: content/user/ui/report/Org-stats.md (hunk 4 above)
Comment (terra4383) [inline content/user/ui/report/Org-stats.md]: The charts are listed out of order and there are references to pieces that you don't see here until you delve into the Org Stats page . "Go to the main Reports page via the User menu to find informational widgets on licensing utilization, breakdowns across various data points for onboarded applications, as well as deployed servers and how they're being utilized. Filters allow you to determine which data you'd like to view at a glance."

File: content/user/ui/report/Org-stats.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+Go go to the Reports page via the User Menu to find information about licensing and breakdowns across various data points for onboarded applications; deployed servers and how they’re being utilized; and licensing to understand utilization and expirations. A collection of filters and charts let you decide which data you want to see and how you want to see it. 
+
+<a href="assets/images/org-stats-reports-page.png" rel="lightbox" title="Organization statistics on Reports page"><img class="thumbnail" src="assets/images/org-stats-reports-page.png"/></a>
+
+In the Licenses chart, view the number of overall licenses for Assess and Protect, as well as the the number of unlicensed applications and servers that exist in your organization. Click the application count to go to the Trial/Unlicensed quick view in the Applications page. 
+
+In the next chart, view active applications. The inner ring designates the breakdown by language; choose the breakdown of the outer ring by selecting **Technology** or **Grade** in the dropdown menu. 
+
+Last, but not least, view the Servers chart. Select **Container** or **Environment** in the dropdown menu to choose how the numbers are analyzed.
```
Comment (terra4383) [inline content/user/ui/report/Org-stats.md]: Telling them to view the Servers chart seems elementary and unnecessary to me. Can we rephrase? "Finally, the servers chart illustrates the breakdown of your deployed servers within the context of either environment or container."

File: content/user/ui/report/Org-stats.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+<a href="assets/images/org-stats-reports-page.png" rel="lightbox" title="Organization statistics on Reports page"><img class="thumbnail" src="assets/images/org-stats-reports-page.png"/></a>
+
+In the Licenses chart, view the number of overall licenses for Assess and Protect, as well as the the number of unlicensed applications and servers that exist in your organization. Click the application count to go to the Trial/Unlicensed quick view in the Applications page. 
+
+In the next chart, view active applications. The inner ring designates the breakdown by language; choose the breakdown of the outer ring by selecting **Technology** or **Grade** in the dropdown menu. 
+
+Last, but not least, view the Servers chart. Select **Container** or **Environment** in the dropdown menu to choose how the numbers are analyzed. 
+
+For more details on each category, select the **View** link under each heading.
```
Comment (terra4383) [inline content/user/ui/report/Org-stats.md]: "To delve into any of this information further, select the View link under each heading."

File: content/user/ui/report/Org-stats.md (hunk 2 above)
Comment (terra4383) [inline content/user/ui/report/Org-stats.md]: I would say "The Licenses tab features an Activity trend chart with license consumption data over the past 12 months."

File: content/user/ui/report/Org-stats.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+Last, but not least, view the Servers chart. Select **Container** or **Environment** in the dropdown menu to choose how the numbers are analyzed. 
+
+For more details on each category, select the **View** link under each heading. 
+
+## Licenses
+
+The Licenses tab features a Consumption trend chart of data for the past 12 months. Hover over a data point on the Assessment or Protection trend lines to see how may licenses were available that month. The thermometer chart below shows the total number of licenses purchased compared to the number being used. The timeline of shows how many licenses are about to expire on given dates. For a different view of the data, take a look at the circular charts with breakdowns by fraction and percentage for Assessment and Protection. 
+
+If you have no Protection licenses, Contrast alerts you to the count of unlicensed servers.
```
Comment (terra4383) [inline content/user/ui/report/Org-stats.md]: Let's not specify Protection, as this empty state could happen in the case of not having Assessment licenses either. Rephrase to be more generic. "If your organization does not currently own one of the license types, Contrast alerts you to the count of relevant unlicensed assets."

File: content/user/ui/report/Org-stats.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+
+<a href="assets/images/org-stats-licenses.png" rel="lightbox" title="Organization statistics for licenses"><img class="thumbnail" src="assets/images/org-stats-licenses.png"/></a>
+
+<!-- Couldn't fit Assessment AND Protection view in browser window-->
+
+## Applications
+
+In the Applications tab, the Status thermometer chart shows the total number of applications broken down by the number that are licensed, unlicensed and archived. Click on the total number of onboarded applications to go to the Applications page for more details on each one. Contrast also remdinds you of how many licenses are available in your organization. 
+
+The circular Language Breakdown chart shows the number of applications by language in the inner band and by Technology or Grade in the outer band. Click the number of active applications to go to the grid in the Applications page. High Risk, Monitored and Expirations snapshots show the number of applications with critical open vulnerabilities, no protection and expiring licenses, respectively. Applications that you added within the last week and applications in an offline server are listed separately in the sidebar.
```
Comment (terra4383) [inline content/user/ui/report/Org-stats.md]: "Click on the active applications count to go to move directly to the Applications page grid......Applications added within the last week as well as applications that reside on an offline server are..."

File: content/user/ui/report/Org-stats.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+
+The circular Language Breakdown chart shows the number of applications by language in the inner band and by Technology or Grade in the outer band. Click the number of active applications to go to the grid in the Applications page. High Risk, Monitored and Expirations snapshots show the number of applications with critical open vulnerabilities, no protection and expiring licenses, respectively. Applications that you added within the last week and applications in an offline server are listed separately in the sidebar. 
+
+<a href="assets/images/org-stats-applications.png" rel="lightbox" title="Organization statistics for applications"><img class="thumbnail" src="assets/images/org-stats-applications.png"/></a>
+
+## Servers  
+
+Switch to the Servers tab to view a thermometer chart that shows the breakdown of all deployed servers by environment. Click on the total number of servers to go to Servers page for more information on each one.
```
Comment (terra4383) [inline content/user/ui/report/Org-stats.md]: "Click on the servers count to go to **the** Servers page."

File: content/user/ui/report/Org-stats.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+
+<a href="assets/images/org-stats-applications.png" rel="lightbox" title="Organization statistics for applications"><img class="thumbnail" src="assets/images/org-stats-applications.png"/></a>
+
+## Servers  
+
+Switch to the Servers tab to view a thermometer chart that shows the breakdown of all deployed servers by environment. Click on the total number of servers to go to Servers page for more information on each one. 
+
+The circular Containers Breakdown chart displays the number of deployed servers for each language in a given environment. Select a different environment in the dropdown menu to update the ring and total number of servers. Click on the total number of servers to go to the grid in the Servers page. Snapshots show servers being Assessed and Protected as well as all servers online compared to the total number of servers in the given environment. The right sidebar includes a list of newly onboarded, offline and deleted servers.
```
Comment (terra4383) [inline content/user/ui/report/Org-stats.md]: Container (singular)

Comment (terra4383) [inline content/user/ui/report/Org-stats.md]: "Clicking on the server count will navigate you to the Servers page with the relevant environment filter applied."

**Comment by terra4383** [comment]
@kaitlynmortimer Lots of comments on this one, and we need to update the screenshots, as they contain some filters that are not going to be implemented. Let me know when you're ready for me to review again and I can assist you with the screenshots on Monday if necessary. THANK YOU!

File: content/user/ui/report/Org-stats.md (hunk 9 above)
Comment (kaitlynmortimer) [inline content/user/ui/report/Org-stats.md]: Rewrite includes Assessment and Protection. Hopefully this version is clearer.

**Comment by kaitlynmortimer** [comment]
Incorporated suggested changes with some variation on wording/phrasing.

**Pull request merged**